config/simon/10002080-13X.xml

```diff
@@ -0,0 +1,354 @@
+      <Item label="78 minutes down time" value="205"/>
+      <Item label="79 minutes down time" value="206"/>
+      <Item label="80 minutes down time" value="207"/>
+      <Item label="81 minutes down time" value="208"/>
+      <Item label="82 minutes down time" value="209"/>
+      <Item label="83 minutes down time" value="210"/>
+      <Item label="84 minutes down time" value="211"/>
+      <Item label="85 minutes down time" value="212"/>
+      <Item label="86 minutes down time" value="213"/>
+      <Item label="87 minutes down time" value="214"/>
+      <Item label="88 minutes down time" value="215"/>
+      <Item label="89 minutes down time" value="216"/>
+      <Item label="90 minutes down time" value="217"/>
+      <Item label="91 minutes down time" value="218"/>
+      <Item label="92 minutes down time" value="219"/>
+      <Item label="93 minutes down time" value="220"/>
+      <Item label="94 minutes down time" value="221"/>
+      <Item label="95 minutes down time" value="222"/>
+      <Item label="96 minutes down time" value="223"/>
+      <Item label="97 minutes down time" value="224"/>
+      <Item label="98 minutes down time" value="225"/>
+      <Item label="99 minutes down time" value="226"/>
+      <Item label="100 minutes down time" value="227"/>
+      <Item label="101 minutes down time" value="228"/>
+      <Item label="102 minutes down time" value="229"/>
+      <Item label="103 minutes down time" value="230"/>
+      <Item label="104 minutes down time" value="231"/>
+      <Item label="105 minutes down time" value="232"/>
+      <Item label="106 minutes down time" value="233"/>
+      <Item label="107 minutes down time" value="234"/>
+      <Item label="108 minutes down time" value="235"/>
+      <Item label="109 minutes down time" value="236"/>
+      <Item label="110 minutes down time" value="237"/>
+      <Item label="111 minutes down time" value="238"/>
+      <Item label="112 minutes down time" value="239"/>
+      <Item label="113 minutes down time" value="240"/>
+      <Item label="114 minutes down time" value="241"/>
+      <Item label="115 minutes down time" value="242"/>
+      <Item label="116 minutes down time" value="243"/>
+      <Item label="117 minutes down time" value="244"/>
+      <Item label="118 minutes down time" value="245"/>
+      <Item label="119 minutes down time" value="246"/>
+      <Item label="120 minutes down time" value="247"/>
+      <Item label="121 minutes down time" value="248"/>
+      <Item label="122 minutes down time" value="249"/>
+      <Item label="123 minutes down time" value="250"/>
+      <Item label="124 minutes down time" value="251"/>
+      <Item label="125 minutes down time" value="252"/>
+      <Item label="126 minutes down time" value="253"/>
+      <Item label="127 minutes down time" value="254"/>
+    </Value>
+
+    <Value genre="config" index="12" label="Behavior of LED in Repose" max="255" min="0" size="1" type="list" value="0">
```

Comment:
maybe a bool instead?

Reply:
I would keep this definition since Simon's 10002041_13X and 10002034_13X (from same family) have the same specification for this values.